package.json

```diff
@@ -27,20 +27,20 @@
   ],
   "dependencies": {
     "@asyncapi/avro-schema-parser": "^1.0.1",
-    "@asyncapi/converter": "^0.11.0",
+    "@asyncapi/converter": "^1.0.0",
     "@asyncapi/openapi-schema-parser": "^2.0.1",
     "@asyncapi/parser": "^1.15.1",
     "@asyncapi/react-component": "^1.0.0-next.38",
     "@asyncapi/specs": "^3.0.0",
     "@headlessui/react": "1.4.1",
     "@hookstate/core": "^3.0.12",
-    "@monaco-editor/react": "^4.2.2",
+    "@monaco-editor/react": "^4.4.5",
     "@semantic-release-plus/docker": "^3.1.1",
     "js-base64": "^3.7.2",
     "js-file-download": "^0.4.12",
     "js-yaml": "^4.1.0",
-    "monaco-editor": "^0.28.1",
-    "monaco-yaml": "^2.5.1",
+    "monaco-editor": "0.28.1",
+    "monaco-yaml": "2.5.1",
```

Comment:
What is the reasoning behind fixing these versions?

Reply:
Higher versions of `monaco-editor` have breaking changes, which are not compatible with given version of `monaco-yaml`. The latest version of `monaco-yaml` supports newer versions of monaco, but here again there are problems related to the worker on the yaml side. Each language in monaco is divided into 2 parts: the code on the main thread side, and the worker, which executes code not on the main thread, but on a separate thread to optimize the process (linting etc). Newer versions of `monaco-yaml` creates this worker differently, and to use it I would have to eject react-scripts and change the webpack. I want to fix this problem but not in this PR. Currently on the hosted studio `monaco-yaml` is not working fully (worker doesn't work) and I would prefer not to wait with these supports and just publish the fixed studio and then deal with dependencies.

Reply:
That makes sense. Thanks or the explanation 👍